kyuubi-server/src/test/scala/org/apache/kyuubi/config/AllKyuubiConfiguration.scala

```diff
@@ -203,8 +203,50 @@ class AllKyuubiConfiguration extends KyuubiFunSuite {
     newOutput += ""
     newOutput += ("Please refer to the Spark official online documentation for" +
       " [SET Command](http://spark.apache.org/docs/latest/sql-ref-syntax-aux-conf-mgmt-set.html)")
+    newOutput += ""
+
+    newOutput += ("## Flink Configurations")
+    newOutput += ""
 
+    newOutput += ("### Via flink-conf.yaml")
     newOutput += ""
+    newOutput += ("Setting them in `$FLINK_HOME/conf/flink-conf.yaml`" +
+      " supplies with default values for SQL engine application." +
+      " Available properties can be found at Flink official online documentation for" +
+      " [Flink Configurations]" +
+      "(https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/)")
+    newOutput += ""
+
+    newOutput += ("### Via kyuubi-defaults.conf")
+    newOutput += ""
+    newOutput += ("Setting them in `$KYUUBI_HOME/conf/kyuubi-defaults.conf`" +
+      " supplies with default values for SQL engine application too." +
+      " These properties of `flink.` prefix will override all settings in" +
+      " `$FLINK_HOME/conf/flink-conf.yaml`.")
```

Comment:
It does not point the additional prefix `flink.`, how about changing it to "You can use properties with the additional prefix `flink.` to override settings in `$FLINK_HOME/conf/flink-conf.yaml`."?